tfrobot/pkg/deployer/deployer.go

```diff
@@ -329,7 +345,11 @@ func getBlockedNodes(groupDeployments groupDeploymentsInfo) []uint64 {
 	return blockedNodes
 }
 
-func buildNetworkDeployment(vm Vms, nodeID uint32, name, solutionType string) workloads.ZNet {
+func buildNetworkDeployment(vm Vms, nodeID uint32, name, solutionType string) workloads.Network {
+	if !vm.PublicIP4 && !vm.Ygg && !vm.Mycelium {
+		vm.Mycelium = true
```

Comment:
we used to log `log.Warn().Str("vms group", vm.Name).Msg("ygg ip, mycelium ip and public IP options are false. Setting ygg IP to true")` if the user didn't request any ip, so can we log similar message but for mycelium?